_includes/layouts/base.html

```diff
@@ -27,7 +27,7 @@
   {% comment %} Hide SVG Sprites {% endcomment %}
   <div style="display: none;">
     {% usa_icons_sprite %}
-    {% uswds_icons_sprite %}
+    {% comment %} {% uswds_icons_sprite %} {% endcomment %}
```

Comment:
Any reason its being comment out?

Reply:
Yes, this was causing a a11y issue because the icons have duplicate `ids`. As far as I can tell only keeping one works but I didn't delete it out yet because I am wondering if something will break. I'll update the comment with a TODO to remove in the future.